lib/api/1.1/southbound/profiles.js

```diff
@@ -47,16 +45,6 @@ function profilesRouterFactory (
                     });
                 }
             })
```

Comment:
Why are we removing this back-off mechanism? Is it not effective/functional and now cruft (was replaced by something else), or just no longer an error condition the workflow engine will generate, so not an issue?